build.sbt

```diff
@@ -34,9 +34,11 @@ ThisBuild / mergifyStewardConfig := Some(
 ThisBuild / mergifyRequiredJobs += "site"
 ThisBuild / mergifyLabelPaths += "docs" -> file("docs")
 
+ThisBuild / resolvers ++= Resolver.sonatypeOssRepos("snapshots")
+
 val catsV = "2.9.0"
-val catsEffectV = "3.3.14"
-val fs2V = "3.3.0"
+val catsEffectV = "3.4.0"
+val fs2V = "3.4-9d4ea04-SNAPSHOT"
```

Comment:
We will need an FS2 release, which is updated to the new `Dispatcher`s in Cats Effect 3.4.0.